editor/modes/visual-editor/style.scss

```diff
@@ -440,3 +440,60 @@
 		box-shadow: none;
 	}
 }
+
+.editor-visual-editor__sibling-inserter {
```

Comment:
I think we should create separate folders for these components to avoid having a giant stylesheet. It's not the first one though, so might be kept for later.

Reply:
>I think we should create separate folders for these components to avoid having a giant stylesheet. It's not the first one though, so might be kept for later.

It might be more a signal that the component itself could do for a reorganization. The way I've approached it so far is: anything other than `./index.js` is a separate but dependent component, and `style.scss` covers both the base and these dependent components. If we're able to factor out some of these subcomponents to be independent, they would have their own stylesheets.

Elsewhere I've taken the approach which might be what you're suggesting, which is to create stylesheets specific to individual components, but I found (and had received feedback) that this became sprawling and disorganized. Example:

https://github.com/Automattic/wp-calypso/tree/master/client/my-sites/media-library